### ISSUE TITLE
Update Turkish email translations

### ISSUE THREAD

**Issue by bobeal** [ozwillo/ozwillo-kernel#9]
As received from Huseyin : 

* Account activation email : 

```
Merhaba….,
 
Bu e-postayı bir Ozwillo hesabı oluşturduğunuz için aldınız.
 
Hesabınızı aktifleştirmek için lütfen aşağıdaki linki tıklayınız:
……
 
Owillo Ekibi
Ozwillo’yu keşfedin!
```

* Account validation email : 

```
Tebrikler ….,
 
Ozwillo hesabınız doğrulandı.
 
Owillo Ekibi
Ozwillo’yu keşfedin!
Ozwillo Kontrol Paneline gidebilirsiniz.
```

* Account pending activation : 

```
Hesap aktive edilmeyi bekliyor.
Hesabınız oluşturuldu ve e-posta adresinizi teyit ederek aktifleştirilmesi gerekiyor.
Size aktivasyon linki içeren bir e-posta gönderdik; gönderilen linki tıklayarak hesabınızı aktifleştirebilirsiniz.
```

**Comment by tbroyer** [comment]
Where should the links be in the following sentences?

> Ozwillo’yu keşfedin!
> Ozwillo Kontrol Paneline gidebilirsiniz.

(specifically the first: should the link be on “Ozwillo” only? on “Ozwillo’yu”? or on the whole line?)

**Comment by tbroyer** [comment]
Fixed by b6cf4e0ba79f63a50268cb34e8aa6fb29d54d665
